L/LibGit2/bundled/patches/libgit2-nfs-permissions.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/odb_pack.c b/src/odb_pack.c
```

Comment:
Can this patch be upstreamed?

Reply:
Let's see: https://github.com/libgit2/libgit2/pull/5935